modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -9,20 +9,24 @@
  */
 def cleanupGit() {
   echo 'Cleaning up git'
-  sh('git clean -fdx')
+  withStatsdTiming("cleanup_git", {
```

Comment:
Just a syntax suggestion, but Groovy gives you a neater way to place the parentheses when the last argument is a closure:

```
withStatsdTiming("cleanup_git") {
  // Function body
}
```

It's also how we call the Jenkins functions like `node` and `stage`.